keyboards/silverbullet44/keymaps/FKML/keymap.c

```diff
@@ -0,0 +1,148 @@
+/* Copyright 2020 FKML
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 3
+
```

Comment:
ごめん、これはこの前の指摘漏れ！
列挙型って言うんだけ？にしとくといいってさ。

```suggestion
enum layer {
    _QWERTY,
    _CURSOL,
    _CALC,
    _ADJUST,
};
```